src/go/k8s/kind.yaml

```diff
@@ -11,11 +11,3 @@ nodes:
   image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
 - role: worker
   image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
-- role: worker
-  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
-- role: worker
-  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
-- role: worker
-  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
-- role: worker
-  image: kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
```

Comment:
How are we doing 4 parallel tests with only five workers?

Reply:
Simple schedule Redpandas on 4 nodes. They don't have anti affinity between separated cluster resources. We have only one test that taints the node https://github.com/redpanda-data/redpanda/tree/2434e021f8685ffce55c99eba7bc3ef4dff790be/src/go/k8s/tests/e2e/node-select-tolerations this can affect 3 node cluster tests, that's I put 4 worker nodes.